src/dali/abstract_pair_converter_plugin.py

```diff
@@ -41,6 +57,37 @@ def __init__(self, historical_price_type: str) -> None:
         result = cast(Dict[AssetPairAndTimestamp, HistoricalBar], load_from_cache(self.cache_key()))
         self.__cache: Dict[AssetPairAndTimestamp, HistoricalBar] = result if result is not None else {}
         self.__historical_price_type: str = historical_price_type
+        self.__session: Session = requests.Session()
+        self.fiat_list: List[str] = []
+
+        try:
+            response: Response = self.__session.get("https://api.exchangerate.host/symbols", timeout=self.__TIMEOUT)
+            # {
+            #     'motd':
+            #         {
+            #             'msg': 'If you or your company ...',
+            #             'url': 'https://exchangerate.host/#/donate'
+            #         },
+            #     'success': True,
+            #     'symbols':
+            #         {
+            #             'AED':
+            #                 {
+            #                     'description': 'United Arab Emirates Dirham',
+            #                     'code': 'AED'
+            #                 },
+            #             ...
+            #         }
+            # }
+            data: Any = response.json()
+            if data[_SUCCESS]:
```

Comment:
We could cache it, but in case a fiat is not found we will need to refresh the cache (in case the list has changed in the backend).

Reply:
I'm not sure where it can 'not find' a fiat. The list is used to check if a conversion is fiat to fiat, and then sends it off to the fiat to fiat function. 

The converter needs the list to know if something is fiat or crypto. If it isn't on this list, the converter assumes it is crypto. Maybe we can have a boolean variable in the ini file, something like `refresh_fiat_list` the user can set to repull the list of fiat. I don't think the list of fiat will change very often.

Reply:
I don't think we should involve the users in cache refreshing, since this is an implementation detail. What if we keep the code as-is: i.e. we don't use cache.py to store the list but we read it once and then keep it in the `fiat_list` variable. How slow is it to read the list once per DaLI run?

Reply:
Refreshing this list takes a minimal amount of time. It is one JSON call and reading in ~100 iso codes. Less than a second. The only reason to cache it is if it is not available (because of server trouble), dali-rp2 can just use the old one. I can't imagine the list of fiat changing often.